src/app/Auth/login-page/login-page.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Store } from '@ngrx/store';
+import { Router } from '@angular/router';
+
+import { AuthService } from '../../services/auth.service';
+import { loginAction } from '../../store/auth.actions';
+import { getIsAuth } from '../../store/auth.reducer';
+
+@Component({
+  selector: 'app-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss']
+})
+export class LoginPageComponent implements OnInit {
+  form: FormGroup;
+  submited = false;
+
+  constructor(private auth: AuthService, private store: Store, private router: Router) {
+  }
+
+  ngOnInit(): void {
+    this.form = new FormGroup({
+      email: new FormControl(null, [Validators.required, Validators.email]),
+      password: new FormControl(null, [Validators.required, Validators.minLength(4)])
+    });
+  }
+
+  onSubmit(): void {
+    this.store.dispatch(loginAction(this.form.value))
+    const isAuth = this.store.select(getIsAuth)
+    isAuth.subscribe(res => {
+      console.log('response login', res)
+      if(res){
+        this.router.navigate(['/forms'])
+      }
+      return
+    })
+  }
```

Comment:
Unsubscribe RxJS Observables In The Angular
https://medium.com/angular-in-depth/the-best-way-to-unsubscribe-rxjs-observable-in-the-angular-applications-d8f9aa42f6a0
```suggestion
    this.store.select(getIsAuth).pipe(
       takeUntil(this.unsubscribe$),
       filter(isAuth => !!isAuth)
    ).subscribe((isAuth: boolean) => this.router.navigate(['/forms']))
  }
```